Formulaic TOS

```diff
@@ -0,0 +1,73 @@
+#Terms of Service - Mozilla Formulaic
```

Comment:
```suggestion
# Mozilla Formulaic Terms of Service
```

There is a missing space, but also this is not the way other documents are titled.

Reply:
Your suggestion looks good to me.